types/webextension-polyfill/namespaces/runtime.d.ts

```diff
@@ -355,18 +487,25 @@ export namespace Runtime {
          * onMessageExternal), if a different extension. Note that extensions cannot send messages to content scripts using this
          * method. To send messages to content scripts, use $(ref:tabs.sendMessage).
          *
-         * @param message
          * @param options Optional.
          */
-        sendMessage(message: any, options?: SendMessageOptionsType): Promise<any>;
+        // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
```

Comment:
Same as above.

Reply:
same reason as for the other sendMessage